Catalogs.API/ContextFactory/CatalogContextFactory.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Catalogs.Infrastructure.Context;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Design;
+
+namespace Catalogs.API.ContextFactory
+{
+    public class CatalogContextFactory : IDesignTimeDbContextFactory<CatalogContext>
+    {
```

Comment:
using IDesignTimeDbContextFactory to create a database context in production code is not a good practice